proto/process/agent.proto

```diff
@@ -10,6 +10,15 @@ import "github.com/DataDog/agent-payload/proto/metrics/agent_payload.proto";
 // Message Types
 //
 
+// ContainerHostType is a enum that represents the type of host detected for container collection.
+// Sometimes containers are running on host-less environment, we'll need additional information to identify them
+// This will help with the host resolution when we resolve container payloads
+enum ContainerHostType {
+	default = 0;
```

Comment:
Should we make `default=1` to make it explicit that we're setting it to that value?  Otherwise we won't be able to distinguish between `default` and "not set", if that matters.

Reply:
I think I consider `default` as "not set". I think you might be confused because `default` could technically mean "regular host". You can't leave out 0 otherwise the code will have a mix of using number and enum which is weird. Shall we give it a more explicit name like `unknown=0` or `unset=0`?

Reply:
I updated the field to be `notSpecified=0`. It won't take `unspecified=0` because a field in `CollectorConnection` has enum of `unspecified` and in protobuf enum fields are top level fields and won't allow duplication.